composer.json

```diff
@@ -7,14 +7,39 @@
 		"bamarni/composer-bin-plugin": "1.8.2",
 		"dealerdirect/phpcodesniffer-composer-installer": "0.7.2",
 		"dms/phpunit-arraysubset-asserts": "0.4.0",
+		"humbug/php-scoper": "0.13.9",
+		"pelago/emogrifier": "6.0.0",
```

Comment:
The `pelago/emogrifier` dependency is intentionally moved to `require-dev`. This ensures that we have only the composer autoloader in the `vendor` folder which autoloads the `vendor_prefixed` classes. The list of dependencies that go in `vendor_prefixed` should be managed in the `php-scoper` config.

Reply:
Isn't `Emgorifier` used in `Sensei\Internal\Emails\Email_Sender`?